app/models/shipit/stack.rb

```diff
@@ -607,5 +608,22 @@ def emit_merge_status_hooks
     def ci_enabled_cache_key
       "stacks:#{id}:ci_enabled"
     end
+
+    def empty_config?
+      cached_deploy_spec.config.empty?
```

Comment:
This seems weird. Should `cached_deploy_spec` be `nil` ?

In which circumstances would `.config` be empty?

Reply:
It’s `{}` until the file has been cached. I guess it’s also that if no file is found, but then it seems to make sense to wait also.

Reply:
Ok my bad. Then it would make sense to expose a more meaningful method on `DeploySpec`.

Accessing its `config` attribute like this isn't really intended.

Reply:
Looks like Ryan has corrected this.